src/app/views/results.py

```diff
@@ -9,8 +9,11 @@ class IndexView(TemplatedView):
 
     def get(self):
         """ GET """
+        user_id = self.request.GET.get('userId', 1)
```

Comment:
I think I might make this so that it prints some kind of copy that talks about the results, and tells you to use your specific link to see them.